frame/nomination-pools/src/lib.rs

```diff
@@ -2115,6 +2397,101 @@ pub mod pallet {
 			ensure!(bonded_pool.can_nominate(&who), Error::<T>::NotNominator);
 			T::Staking::chill(&bonded_pool.bonded_account())
 		}
+
+		/// Set the commission of a pool.
+		///
+		/// The dispatch origin of this call must be signed by the `root` role of
```

Comment:
are these 100 line width?

Reply:
It was on 80, amended to 100. Change is reflected in the latest commit.